<launch>
	<arg
		name="device_name"
		default="inhouse_autosampler"/>
	<arg
		name="connection_mode"
		default="serial"/>
	<arg
		name="address"
		default="None"/>
	<arg
		name="port"
		default="/dev/ttyUSB0"/>
	<arg
		name="simulation"
		default="False"/>
	<node
		name="inhouse_autosampler"
		pkg="roslabware_drivers"
		type="inhouse_autosampler"
		args="
			-d $(arg device_name)
			-c $(arg connection_mode)
			-add $(arg address)
			-p $(arg port)
			-l $(arg simulation)
			"
		output="screen"/>
</launch>
